packages/material-ui/src/useAutocomplete/useAutocomplete.test.js

```diff
@@ -255,11 +255,9 @@ describe('useAutocomplete', () => {
     };
 
     const devErrorMessages = [
-      !React.version.startsWith('18') &&
-        "Error: Uncaught [TypeError: Cannot read property 'removeAttribute' of null]",
```

Comment:
This was actually a bug reported in https://github.com/facebook/react/issues/21712